lib/compat/wordpress-6.5/class-wp-navigation-block-renderer.php

```diff
@@ -251,24 +253,29 @@ private static function get_inner_blocks( $attributes, $block ) {
 		}
 
 		// Load inner blocks from the navigation post.
-		if ( array_key_exists( 'ref', $attributes ) ) {
+		if ( $navigation_post_id ) {
 			$inner_blocks = static::get_inner_blocks_from_navigation_post( $attributes );
 		}
 
 		// If there are no inner blocks then fallback to rendering an appropriate fallback.
 		if ( empty( $inner_blocks ) ) {
-			$inner_blocks = static::get_inner_blocks_from_fallback( $attributes );
+			$inner_blocks            = static::get_inner_blocks_from_fallback( $attributes );
+			$fallback_naviation_post = gutenberg_block_core_navigation_get_fallback_post();
+
+			if ( $fallback_naviation_post ) {
+				$navigation_post_id = $fallback_naviation_post->ID;
+			}
```

Comment:
```suggestion
			$fallback_navigation_post = gutenberg_block_core_navigation_get_fallback_post();

			if ( $fallback_navigation_post ) {
				$navigation_post_id = $fallback_navigation_post->ID;
			}
```

Reply:
Nice catch, fixed in https://github.com/WordPress/gutenberg/pull/57754/commits/19cc44c0ddc21e881dc971d26f4bbb74439fe612